tests/lib/fake_iam.py

```diff
@@ -26,7 +26,7 @@ def expect_upload_server_certificate(
                 "ServerCertificateId": "FAKE_CERT_ID_XXXXXXXX",
                 "Path": path,
                 "ServerCertificateName": name,
-                "Arn": f"arn:aws:iam::000000000000:server-certificate{path}/{name}",
+                "Arn": f"arn:aws:iam::000000000000:server-certificate{path}{name}",
```

Comment:
The forward slash isn't needed here?  I'm guessing it's included in one of the variables being interpolated in?

Reply:
yeah, this was also a nit-picky thing I just stumbled on while testing. No current functionality makes any assumptions about what the ARN looks like, so this really just makes the error look more realistic if/when your tests fail

Reply:
Awesome, thanks!